modules/statics/src/account.ts

```diff
@@ -71,10 +71,10 @@ export class AccountCoinToken extends AccountCoin {
 }
 
 /**
- * ERC 20 is a token standard for the Ethereum blockchain. They are similar to other account coins, but have a
- * contract address property which identifies the smart contract which defines the token.
+ * Some blockchains support tokens which are defined by an address at which they have a smart contract deployed.
+ * Examples are ERC20 tokens, and the equivalent on other chains.
  */
-export class Erc20Coin extends AccountCoinToken {
+export class ContractAddressDefinedToken extends AccountCoinToken {
```

Comment:
Added this for the following reasons:
- deduplicate code for all of these ERC20-esque coins
- allow `instanceof` checks in an intuitive way (i.e. Celo ERC20 coins should *not* be a subclass of ETH ERC20 coins)
- Allow `instanceof ContractAddressDefinedToken` checks if a user of this library wants to see if a given coin has `contractAddress` field since `if (coin.contractAddress)` doesn't work in typescript clients because the output of `CoinMap` is just `BaseCoin`